stix_shifter_modules/qradar/stix_translation/query_constructor.py

```diff
@@ -79,8 +79,7 @@ def _format_equality(value) -> str:
 
     @staticmethod
     def _format_like(value) -> str:
-        value = "'%{value}%'".format(value=value)
-        return AqlQueryStringPatternTranslator._escape_value(value)
+        return "'%{value}%'".format(value=value)
```

Comment:
it looks good. I found an existing issue even though I think, it doesn't affect the aql query. If LIKE operator in STIX Pattern contain start and end `%` character than the translated query end up having two `%` character which is unnecessary. We can check the start and end % char and then format the string.

Reply:
I checked with QRadar, if you use LIKE the extra '%' is ignored ('%%xx%%' treated like '%xx%')

Reply:
Right. I'm merging this as it is. if we need we can handle extra % later.